acme/acme/messages.py

```diff
@@ -172,8 +172,9 @@ class Directory(jose.JSONDeSerializable):
     class Meta(jose.JSONObjectWithFields):
         """Directory Meta."""
         terms_of_service = jose.Field('terms-of-service', omitempty=True)
+        terms_of_service_v2 = jose.Field('termsOfService', omitempty=True)
         website = jose.Field('website', omitempty=True)
-        caa_identities = jose.Field('caa-identities', omitempty=True)
+        caa_identities = jose.Field('caaIdentities', omitempty=True)
```

Comment:
Is this what it is in v2? We presumably still need the v1 version around, no?

Reply:
This is what it is in v2. Boulder doesn't set this field in either v1 or v2, so we can be confident no one is relying on it and we can change it.

Reply:
Are you planning to ever set it for v1 servers?

Reply:
Nope.